code/modules/spells/targeted/recall.dm

```diff
@@ -102,16 +119,24 @@
 	return 1
 
 /spell/targeted/bound_object/empower_spell()
-	spell_levels[Sp_POWER]++
-	allow_anchored = 1
-
-	var/upgrade_desc = "You have reduced the restrictions on your binding."
-
+	var/upgrade_desc
+	if(spell_levels[Sp_POWER] == 0)
+		spell_levels[Sp_POWER]++
+		allow_anchored = 1
+		upgrade_desc = "You have reduced the restrictions on your binding."
+	else
+		spell_levels[Sp_POWER]++
+		upgrade_desc = "You can now bind far more destructive objects to yourself."
 	return upgrade_desc
 
 /spell/targeted/bound_object/get_upgrade_info(upgrade_type, level)
 	if(upgrade_type == Sp_POWER)
-		return "Increases your binding skill, allowing otherwise immobile structures and machines to be moved."
+		if(spell_levels[Sp_POWER] == 0)
+			return "Increases your binding skill, allowing otherwise immobile structures and machines to be moved."
+		if(spell_levels[Sp_POWER] == 1)
+			return "Further increases your binding skill, allowing you to bind certain incredibly destructive objects."
```

Comment:
Given that there are only two things on the list, I feel you should make this message less ambiguous. What does this mean to the average user? Fireaxes? Bombs? Plasma canisters?

They should know what they're buying access to.

Reply:
It was meant to be more ambiguous because it's a list that could have new things added to it in the future, but I can change it to mention the gravitational singularity and supermatter specifically.

Reply:
I understand that line of thinking. If you wanted to make it perfectly futureproof, you can actually have it generate an english_list of all objects this makes bindable. You could make a new helper for this.

```
//Convert a list of types into a list of object names
/proc/english_list_of_types(var/list/L)
  var/list/names = list()
   for(var/P in L)
      if(!ispath(P))
         continue
      names += P::name
   return english_list(names)
```

Once implemented, this would allow you to make the description say...
"Allows you to bind the supermatter shard and gravitational singularity."
But it would also automatically expand every time you add something to it, e.g.:
"Allows you to bind the supermatter shard, gravitational singularity, and blob."

Reply:
![Improvement](https://github.com/vgstation-coders/vgstation13/assets/41342767/1b41e242-cdef-4677-ba4b-2750b5899efc)